capacity-mode-evaluator/src/metrics_estimates.py

```diff
@@ -0,0 +1,185 @@
+import pandas as pd
+from datetime import datetime, timedelta, date
+
+
+def maxA(i, j):
+    if i > j:
```

Comment:
It is just a very minor change, but I would love to see this as one liner.

`return i if i>j else j`